nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexMojo.java

```diff
@@ -65,6 +66,11 @@ public class NonDexMojo extends AbstractNonDexMojo {
     private List<NonDexSurefireExecution> executions = new LinkedList<>();
     private ArrayList<CleanSurefireExecution> executionsWithoutShuffling =
             new ArrayList<CleanSurefireExecution>();
+    private Path runFilePath = null;
+
+    public void setFilePath(Path value) {
+        this.runFilePath = this.runFilePath == null ? value : this.runFilePath;
```

Comment:
This may have been easier to read with an `if`, and I'm not sure how this entire thing works with threads, but let me accept.